chart/values.yaml

```diff
@@ -140,9 +140,38 @@ updateStrategy:
 volumeMounts:
   cache:
     mountPath: /var/cache/nginx
+  custom-networks-config:
+    mountPath: /app/networks-config.json
+    subPath: networks-config.json
 
 # Volume mounts to add to the container. The key is the volume name and the value is the volume definition. Evaluated as a template.
 volumes:
   cache:
     emptyDir: {}
 
+  custom-networks-config:
+    configMap:
+      name: hedera-explorer-config
+      
+
+config: |
+     [
```

Comment:
I would've expected two spaces for indentation for the config but count 5. Not sure if it causes templating problems though.